website/templates/emails/pending_retraction.txt.mako

```diff
@@ -0,0 +1,14 @@
+Hello ${user.fullname},
```

Comment:
@bnosek @saradbowman And this is the email template for retraction requests sent to project admins